.github/workflows/build_wheels.yml

```diff
@@ -21,14 +21,14 @@ jobs:
       - uses: actions/setup-python@v2
         name: Install Python
         with:
-          python-version: '3.10'
+          python-version: '3.11'
```

Comment:
No need to change this (since this is only the environment for calling cibuildwheel). Better keep a stable, fixed tooling.